quicklogic/common/utils/eos_s3_iomux_config.py

```diff
@@ -239,10 +246,17 @@ def main():
     # Convert the config to IOMUX register content
     iomux_regs = generate_iomux_register_content(config)
 
-    # Output JLink commands
     print("")
-    for adr in sorted(iomux_regs.keys()):
-        print("w4 0x{:08X} 0x{:08X}".format(adr, iomux_regs[adr]))
+    if args.output_format == "openocd":
+        # Output openOCD process
+        print('proc setup_iomux {} {')
+        for adr in sorted(iomux_regs.keys()):
+            print("    mww 0x{:08X} 0x{:08X}".format(adr, iomux_regs[adr]))
+        print('}')
+    else:
```

Comment:
I'd use here `elif args.output_format == "jlink"` and then later print some error message under the final `else`